### DISCUSSION TITLE
Add Condition in custom perspective for orders and order items

### DISCUSSION THREAD

**Discussion by shubhangi92** [pimcore/pimcore#8479] [Q&A / Support]
Hi Team,

I am looking for solution of a problem related to custom perspectives,  There are classes ie. orders and order items (parent is orders) now i want all orders and inside order items which belongs to specific store. Association of stores with order items .
With the following code snippet i got all orders and their order items but i need to add condition like orderitem.store = 'somestore or store id" so that store managers can see only their order items when they open order.

         "treetype" => "object",
            "name" => "Order Management",
            "condition" => NULL,
            'iconCls' => 'pimcore_icon_perspective',
            "icon" => "/bundles/app/assets/images/icon/orders.png",
            "id" => 6,
            "rootfolder" => "/Orders",
            "showroot" => FALSE,
           // "classes" => "AP",
            "position" => "left",
            "sort" => "4",
            "expanded" => TRUE

Look forward for the solution.

**Comment by neha04** [comment]
Have you tried using joins as given in the documentaion?
https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Class_Settings/Custom_Views.html#page_Advanced-Features-Configurations
